Tools/upload_log.py

```diff
@@ -38,12 +38,7 @@ def ask_value(text, default=None):
     if default != None:
         ask_string += ' (Press ENTER to use ' + default + ')'
     ask_string += ': '
-
-    if sys.version_info[0] < 3:
-        ret = raw_input(ask_string)
-    else:
-        ret = input(ask_string)
-
+    ret = input(ask_string).strip()
```

Comment:
@cclauss doesn't this require the import magic for `input`?

Reply:
__from six.moves import input__ is all the _magic_ that is required.

Reply:
aha, thanks!